hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/TarContainerPacker.java

```diff
@@ -292,15 +272,17 @@ private static ArchiveOutputStream tar(OutputStream output) {
     return new TarArchiveOutputStream(output);
   }
 
-  private InputStream decompress(InputStream input)
+  public InputStream decompress(InputStream input)
```

Comment:
If the scope is only opened up for tests, I think you don't need to make it public.
Make it package-private is good enough, and use VisibleForTesting annotation to flag that the scope is only opened up for tests.
 ```suggestion
  @VisibleForTesting
  InputStream decompress(InputStream input)
```

Reply:
finished